hooks/hooks.py

```diff
@@ -32,8 +32,7 @@ def config_changed():
     CHARM_DIR = os.environ.get('CHARM_DIR', '')
     kubernetes_file = os.path.join(CHARM_DIR, 'files', kubernetes_tar_file)
     installer = KubernetesInstaller(arch, version, kubernetes_file)
-    # Install the kubernetes binary files in the /opt/kubernetes/bin directory.
-    installer.install('/opt/kubernetes/bin')
+    installer.install('/opt/kubernetes')
```

Comment:
Thanks for the review.  I will add back in the comment describing what the install method does.